lms/djangoapps/teams/static/teams/js/views/team_join.js

```diff
@@ -0,0 +1,132 @@
+;(function (define) {
+'use strict';
+
+define(['backbone',
+        'underscore',
+        'gettext',
+        'text!teams/templates/team-join.underscore'],
+       function (Backbone, _, gettext, team_join_template) {
+           return Backbone.View.extend({
+
+               errorMessage: gettext("An error occurred. Please try again."),
+               alreadyMemberMessage: gettext("You are already in a team"),
+               teamFullMessage: gettext("Team is full"),
```

Comment:
"You are already in a team" -> "You already belong to another team."
"Team is full" -> "This team is full."